RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+With this release in Chalice's `GraphQLView` the parameter `render_graphiql` is now deprecated and has been replaced with `graphiq`. It will be removed in releases of Strawberry made after 2022-02-22.
```

Comment:
> Strawberry made after 2022-02-22.

I'll start a discussion on discord about using dates for deprecation warning in the next couple of days as we have two options:

1. remove things after date X
2. remove things after release X

Reply:
Should we link to this PR (and the one that will close it) in the comment?

Reply:
```suggestion
With this release in Chalice's `GraphQLView` the parameter `render_graphiql` is now
deprecated and has been replaced with `graphiql`. It will be removed in releases of
Strawberry made after 2022-02-22.
```
Note the `graphq` -> `graphiql` typo adjustment, but I also made it wrap so it was easier for me to read :grin: